tailwind.config.js

```diff
@@ -10,3 +10,18 @@ module.exports = {
   variants: {},
   plugins: [],
 };
+module.exports = {
+  theme: {
+    borderRadius: {
+      mm: '20px'
+    },
+    boxShadow: {
+      ps: '28px 2px 18px 10px #101010'
+    },
+    margin: {
+               s: '-28%',
```

Comment:
for negative margins, check out [this article](https://tailwindcss.com/docs/margin#negative-values), and I would say avoid percentages when it comes to margins because it breaks so easily. Try to find values that work from tailwind predefined values.